### ISSUE TITLE
sphinx-build.exe missing conf.py

### ISSUE THREAD

**Issue by piousminion** [Sebass13/aiorcon#2]
I'm using:
Python: 3.7.4
aiorcon: 0.6.8 (9a02652?)
OS: Windows 10

I'm trying to make a simple script to talk to a Rust server to monitor it's status.  I don't see any examples for your code and I'm no python veteran.  When attempting to build the docs via Sphinx, which I never heard of until today, I get the following error:

```
PS C:\Users\clay\Downloads\aiorcon-master\docs> sphinx-build.exe . _build

Application error:
config directory doesn't contain a conf.py file (.)
PS C:\Users\clay\Downloads\aiorcon-master\docs>
```

Any assistance you could provide would be greatly appreciated. :)

**Comment by 991jo** [comment]
Yes, you are right, Sphinx is missing that file. Also the documentation is missing in general.
I will take a look at this.

**Comment by skmendez** [comment]
I'm no longer active with this project, but I wrote a high level overview for another user a few weeks ago, so I'll just paste that in here:

The docs didn’t generate because I never got around to actually creating them, haha. In short, you should create a RCON instance using the async class method RCON.create, where you pass in the host, port, password, and optionally the asyncio event loop. See here: 
https://github.com/skmendez/aiorcon/blob/master/aiorcon/rcon.py#L10-L12
Depending on the game server you are using RCON for, you may have to disable multiple_packet. It’s probably something you can look up. I know Valve games like CS:GO support it, but I believe some users had trouble with games like ARK. There’s also an optional auto_reconnect_cb that can be called whenever a reconnect attempt happens. The RCON class really just is a wrapper for the RCONProtocol which subclasses asyncio.Protocol. This runs the connection and authentication procedures. The execute method of the RCONProtocol class is what you supply your command to; you can also do this by simply calling an instantiated RCON instance, as the __call__ method for an RCON instance essentially wraps the execute method of its internal RCONProtocol, see here:
https://github.com/skmendez/aiorcon/blob/master/aiorcon/rcon.py#L60-L74
If you need any more help, please let me know. Also, if you would like to contribute better documentation to this project, feel free to open a PR; I don’t have the time to spend working on this project which I know longer use, but I’d be happy to help others grow it still.
